doc-content/enterprise-only/release-notes/rn-740-fixed-issues-ref.adoc

```diff
@@ -0,0 +1,125 @@
+* All line separators in RHS in a spreadsheet are escaped [https://issues.jboss.org/browse/RHDM-903[RHDM-903]]
+//* Custom data objects are not shown in guided rules  [https://issues.jboss.org/browse/RHDM-884[RHDM-884]]
+* Drools Test Scenario Audit Log content does not fit in the table view  [https://issues.jboss.org/browse/RHDM-911[RHDM-911]]
+* Cannot resize a column in a guided rule template   [https://issues.jboss.org/browse/RHDM-926[RHDM-926]]
+* Localize the  `library.properties` file  [https://issues.jboss.org/browse/RHDM-688[RHDM-688]]
+* Read-only user can delete fields in Data Object  [https://issues.jboss.org/browse/RHDM-756[RHDM-756]]
+
+== Business Optimizer
+* Employee Rostering shows an exception message when deleting an entity that is already referenced from some other entity  [https://issues.jboss.org/browse/RHDM-919[RHDM-919]]
+* Unable to deploy the out-of-the-box Employee Roster starter application WAR file  [https://issues.jboss.org/browse/RHDM-923[RHDM-923]]
+
+== Decision engine
+* A build error occors with executable-model, nested property with double-bytes property name  [https://issues.jboss.org/browse/RHDM-899[RHDM-899]]
+* `MVELConcequenceBuilder` treats the second occurance of the '#' character within a string incorrectly as a comment [https://issues.jboss.org/browse/RHDM-855[RHDM-855]]
+* Missing node sharing in executable model when model sources are generated in multiple stages [https://issues.jboss.org/browse/RHDM-955[RHDM-955]]
+* NPE when creating a KIE container with a guided decision table and not having `drools-workbench-models-guided-dtable` on classpath  [https://issues.jboss.org/browse/RHDM-57[RHDM-57]]
+* An inaccurate warning appears while building a KJAR [https://issues.jboss.org/browse/RHDM-962[RHDM-962]]
+* Parser error with executable-model on map literal expression in constraint [https://issues.jboss.org/browse/RHDM-897[RHDM-897]]
+* `SAXNotRecognizedException` is thrown when using Kie Builder on IBM Java  [https://issues.jboss.org/browse/RHDM-851[RHDM-851]]
+* ServiceDiscovery does not work with Fuse [https://issues.jboss.org/browse/RHDM-915[RHDM-915]]
+* An inappropriate warning appears while building a KJAR [https://issues.jboss.org/browse/RHDM-962[RHDM-962]]
+* Rules in a spreadsheet do not behave the same way with DRL  [https://issues.jboss.org/browse/RHDM-924[RHDM-924]]
+
+== DMN
+* DMN large memory allocation bug [https://issues.jboss.org/browse/RHDM-907[RHDM-907]]
+* Laptop arrow keys do not move selection through all of the options in a dropdown menu  [https://issues.jboss.org/browse/RHDM-912[RHDM-912]]
+
+== Migration tool
+* Error upgrading from {PRODUCT_DM} 7.2.1 to 7.3.0 using the migration tool `rhdm-7.3.0-update`  [https://issues.jboss.org/browse/RHDM-944[RHDM-944]]
+
+== OpenShift
+* User password is exposed in the Readiness and Liveness probes  [https://issues.jboss.org/browse/RHDM-848[RHDM-848]]
```

Comment:
In the PAM version of the RN, this is listed under "Business Central" and worded slightly differently, as follows: "User password is exposed in the Readiness probe and Liveness probe" Just for what it's worth. I like the PAM version because it clarifies that they are two probes.